### PR TITLE
Optim package docstring fix

### LINKED ISSUE #112593: Fix docstring errors in nadam.py, radam.py, sgd.py, anomaly_mode.py, rprop.py, __init__.py, swa_utils.py, rmsprop.py, optimizer.py, lr_scheduler.py
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `_initial_step`, **Line**: 82, **Description**: First line should end with a period (not 'p')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `state_dict`, **Line**: 88, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `load_state_dict`, **Line**: 96, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `get_last_lr`, **Line**: 105, **Description**: One-line docstring should fit on one line with quotes (found 2)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `get_last_lr`, **Line**: 105, **Description**: No whitespaces allowed surrounding docstring text
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `print_lr`, **Line**: 114, **Description**: One-line docstring should fit on one line with quotes (found 2)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `LambdaLR`, **Line**: 187, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `LambdaLR`, **Line**: 187, **Description**: First line should end with a period (not 'r')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `state_dict`, **Line**: 224, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `state_dict`, **Line**: 224, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `load_state_dict`, **Line**: 244, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `load_state_dict`, **Line**: 244, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `MultiplicativeLR`, **Line**: 273, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `MultiplicativeLR`, **Line**: 273, **Description**: First line should end with a period (not 'n')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `state_dict`, **Line**: 308, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `load_state_dict`, **Line**: 325, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `StepLR`, **Line**: 354, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `StepLR`, **Line**: 354, **Description**: First line should end with a period (not 'y')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `MultiStepLR`, **Line**: 403, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `MultiStepLR`, **Line**: 403, **Description**: First line should end with a period (not 'e')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `ConstantLR`, **Line**: 452, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `ConstantLR`, **Line**: 452, **Description**: First line should end with a period (not 'e')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `LinearLR`, **Line**: 510, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `LinearLR`, **Line**: 510, **Description**: First line should end with a period (not 'l')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `ExponentialLR`, **Line**: 578, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `SequentialLR`, **Line**: 609, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `SequentialLR`, **Line**: 609, **Description**: First line should end with a period (not 'g')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `state_dict`, **Line**: 686, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `load_state_dict`, **Line**: 701, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `PolynomialLR`, **Line**: 718, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `PolynomialLR`, **Line**: 718, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `PolynomialLR`, **Line**: 718, **Description**: First line should end with a period (not 'n')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `CosineAnnealingLR`, **Line**: 768, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `CosineAnnealingLR`, **Line**: 768, **Description**: First line should end with a period (not 'g')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `ChainedScheduler`, **Line**: 841, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `ChainedScheduler`, **Line**: 841, **Description**: First line should end with a period (not 'g')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `state_dict`, **Line**: 882, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `load_state_dict`, **Line**: 897, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `ReduceLROnPlateau`, **Line**: 914, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `_reset`, **Line**: 1002, **Description**: First line should be in imperative mood (perhaps 'Reset', not 'Resets')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `CyclicLR`, **Line**: 1088, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `CyclicLR`, **Line**: 1088, **Description**: First line should end with a period (not 'o')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `get_lr`, **Line**: 1285, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `get_lr`, **Line**: 1285, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `get_lr`, **Line**: 1285, **Description**: First line should end with a period (not 's')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `get_lr`, **Line**: 1285, **Description**: First line should be in imperative mood (perhaps 'Calculate', not 'Calculates')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `CosineAnnealingWarmRestarts`, **Line**: 1339, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `CosineAnnealingWarmRestarts`, **Line**: 1339, **Description**: First line should end with a period (not 'g')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `step`, **Line**: 1390, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `step`, **Line**: 1390, **Description**: First line should end with a period (not 'e')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `OneCycleLR`, **Line**: 1464, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `OneCycleLR`, **Line**: 1464, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `OneCycleLR`, **Line**: 1464, **Description**: First line should end with a period (not 'e')
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `_annealing_cos`, **Line**: 1695, **Description**: Use """triple double quotes""" (found "-quotes)
- **File**: `torch/optim/lr_scheduler.py`, **Entity**: `_annealing_linear`, **Line**: 1701, **Description**: Use """triple double quotes""" (found "-quotes)
- **File**: `torch/optim/nadam.py`, **Entity**: `step`, **Line**: 70, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/optim/nadam.py`, **Entity**: `nadam`, **Line**: 176, **Description**: No blank lines allowed after function docstring (found 2)
- **File**: `torch/optim/optimizer.py`, **Entity**: `_RequiredParameter`, **Line**: 25, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_optimizer_step_pre_hook`, **Line**: 143, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_optimizer_step_pre_hook`, **Line**: 143, **Description**: First line should end with a period (not 'g')
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_optimizer_step_post_hook`, **Line**: 162, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_optimizer_step_post_hook`, **Line**: 162, **Description**: First line should end with a period (not 'g')
- **File**: `torch/optim/optimizer.py`, **Entity**: `_group_tensors_by_device_and_dtype`, **Line**: 323, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/optimizer.py`, **Entity**: `_group_tensors_by_device_and_dtype`, **Line**: 323, **Description**: Multi-line docstring closing quotes should be on a separate line
- **File**: `torch/optim/optimizer.py`, **Entity**: `_group_tensors_by_device_and_dtype`, **Line**: 323, **Description**: First line should be in imperative mood (perhaps 'Group', not 'Groups')
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_step_pre_hook`, **Line**: 338, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_step_pre_hook`, **Line**: 338, **Description**: First line should end with a period (not 'e')
- **File**: `torch/optim/optimizer.py`, **Entity**: `register_step_post_hook`, **Line**: 360, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/optimizer.py`, **Entity**: `state_dict`, **Line**: 380, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/optim/optimizer.py`, **Entity**: `load_state_dict`, **Line**: 432, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/optim/optimizer.py`, **Entity**: `zero_grad`, **Line**: 488, **Description**: First line should be in imperative mood (perhaps 'Reset', not 'Resets')
- **File**: `torch/optim/optimizer.py`, **Entity**: `step`, **Line**: 529, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/optim/optimizer.py`, **Entity**: `step`, **Line**: 529, **Description**: First line should not be the function's "signature"
- **File**: `torch/optim/radam.py`, **Entity**: `step`, **Line**: 84, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/optim/radam.py`, **Entity**: `radam`, **Line**: 200, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/rmsprop.py`, **Entity**: `step`, **Line**: 97, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/optim/rmsprop.py`, **Entity**: `rmsprop`, **Line**: 217, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/rmsprop.py`, **Entity**: `rmsprop`, **Line**: 217, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/rprop.py`, **Entity**: `step`, **Line**: 82, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/optim/rprop.py`, **Entity**: `rprop`, **Line**: 188, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/sgd.py`, **Entity**: `step`, **Line**: 57, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/optim/sgd.py`, **Entity**: `sgd`, **Line**: 200, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/optim/swa_utils.py`, **Entity**: `AveragedModel`, **Line**: 67, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/optim/swa_utils.py`, **Entity**: `AveragedModel`, **Line**: 67, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/optim/swa_utils.py`, **Entity**: `AveragedModel`, **Line**: 67, **Description**: First line should end with a period (not 'd')
- **File**: `torch/optim/swa_utils.py`, **Entity**: `update_bn`, **Line**: 227, **Description**: First line should be in imperative mood (perhaps 'Update', not 'Updates')
- **File**: `torch/optim/swa_utils.py`, **Entity**: `SWALR`, **Line**: 280, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/optim/_multi_tensor/__init__.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/autograd/__init__.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/autograd/__init__.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not 'c')
- **File**: `torch/autograd/__init__.py`, **Entity**: `backward`, **Line**: 118, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/autograd/__init__.py`, **Entity**: `backward`, **Line**: 118, **Description**: First line should end with a period (not 'h')
- **File**: `torch/autograd/__init__.py`, **Entity**: `backward`, **Line**: 118, **Description**: First line should be in imperative mood (perhaps 'Compute', not 'Computes')
- **File**: `torch/autograd/__init__.py`, **Entity**: `grad`, **Line**: 219, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/autograd/__init__.py`, **Entity**: `grad`, **Line**: 219, **Description**: First line should end with a period (not 'o')
- **File**: `torch/autograd/__init__.py`, **Entity**: `grad`, **Line**: 219, **Description**: First line should be in imperative mood (perhaps 'Compute', not 'Computes')
- **File**: `torch/autograd/anomaly_mode.py`, **Entity**: `detect_anomaly`, **Line**: 10, **Description**: No blank lines allowed between a section header and its content ('Example')

cc @brycebortree @carljparker

### PR DESCRIPTION
**Pull request by spzala**
Fix docstrings in various files in optim package. This is a last remaining fix for the issue #112593

The fix can be verified by running pydocstyle path-to-file --count

Fixes #112593

Related #128248

cc @svekars @brycebortree

### CODE REVIEW

**Comment by spzala** [comment]
cc @svekars @janeyx99 I have addressed Optim related comments from https://github.com/pytorch/pytorch/pull/128248 Thanks!

File: torch/optim/swa_utils.py [hunk 1]
```diff
@@ -43,6 +46,8 @@ def ema_update(ema_param_list: PARAM_LIST, current_param_list: PARAM_LIST, _):
 
 
 def get_swa_multi_avg_fn():
+    """Receive stochastic weight average."""
```
[the remark this answers is not in the document]
Comment (janeyx99) [reply torch/optim/swa_utils.py:49]: Wait this one should be across multiple params.

File: torch/optim/swa_utils.py [hunk 2]
```diff
@@ -81,6 +88,8 @@ def ema_update(ema_param: Tensor, current_param: Tensor, num_averaged):
 
 
 def get_swa_avg_fn():
+    """Receive stochastic weight average."""
```
Comment (janeyx99) [inline torch/optim/swa_utils.py:91]: and this one should be across a single param.

File: torch/optim/swa_utils.py [hunk 3]
```diff
@@ -425,6 +436,7 @@ def _get_initial_lr(lr, swa_lr, alpha):
         return (lr - alpha * swa_lr) / (1 - alpha)
 
     def get_lr(self):
+        """Compute learning rate."""
```
Comment (janeyx99) [inline torch/optim/swa_utils.py:439]:
```suggestion
        """Get learning rate."""
```

**Review by janeyx99 — COMMENTED** [review]
Sorry, computer forced restart in the middle of my review. Left the rest now.

File: torch/optim/swa_utils.py (hunk 2 above)
Comment (spzala) [reply torch/optim/swa_utils.py:91]: Sure @janeyx99 Thanks so much for the quick review and suggestions! I will be updating the PR soon.

**Comment by spzala** [comment]
> Sorry, computer forced restart in the middle of my review. Left the rest now.

:) No problem at all, and thank you so much for the review suggestions!!! I have updated the PR accordingly @janeyx99

File: torch/optim/swa_utils.py [hunk 4]
```diff
@@ -43,6 +46,8 @@ def ema_update(ema_param_list: PARAM_LIST, current_param_list: PARAM_LIST, _):
 
 
 def get_swa_multi_avg_fn():
+    """Get the function applying stochastic weight average (SWA) across a multiple params."""
```
Comment (janeyx99) [inline torch/optim/swa_utils.py:49]:
```suggestion
    """Get the function applying stochastic weight average (SWA) across multiple params."""
```

**Review by janeyx99 — APPROVED** [review]

File: torch/optim/swa_utils.py (hunk 4 above)
Comment (spzala) [reply torch/optim/swa_utils.py:49]: err, my bad sorry! Thanks for catching @janeyx99

Comment (spzala) [reply torch/optim/swa_utils.py]: Addressed your comment @janeyx99 Thanks!

**Comment by spzala** [comment]
Thank you @janeyx99 !!

**Pull request closed without merging**